easybuild/easyconfigs/f/freetype/freetype-2.6.2-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,21 @@
+name = 'freetype'
+version = '2.6.2'
+
+homepage = 'http://freetype.org'
+description = """FreeType 2 is a software font engine that is designed to be small, efficient, highly customizable, and
+ portable while capable of producing high-quality output (glyph images). It can be used in graphics libraries, display
+ servers, font conversion tools, text image generation tools, and many other products as well."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+
+source_urls = [GNU_SAVANNAH_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('libpng', '1.6.21')]
+
+sanity_check_paths = {
+    'files': ['bin/freetype-config', 'lib/libfreetype.a', 'lib/libfreetype.so', 'lib/pkgconfig/freetype2.pc'],
```

Comment:
please use `'lib/libfreetype.%s' % SHLIB_EXT` rather than hardcoding `.so`, and then wrap the `.pc` to the next line (line gets too long, I think)

cfr. existing freetype easyconfigs in `develop`